### ISSUE TITLE
setOnMarkerDragListener does not report marker drag event

### ISSUE THREAD

**Issue by gpopac** [googlemaps/android-maps-utils#1329]
I have trouble implementing marker dragging with android-maps-utils, despite following the guideline. The marker is draggable, but dragging it on the map does not trigger onMarkerDrag, onMarkerDragEnd or onMarkerDragStart.

I am using KmlLayer and GeoJsonLayer to add custom overlay on the map and simple marker only (not clusters). For more details the code of the Activity is available here: https://github.com/Biologer/Biologer-Android/blob/master/app/src/main/java/org/biologer/biologer/gui/MapActivity.java

In summary I used:
```
Marker marker;
MarkerManager markerManager = new MarkerManager(googleMap);
MarkerManager.Collection markerCollection = markerManager.newCollection();
MarkerOptions markerOptions = new MarkerOptions().position(latLong).draggable(true);
marker = markerCollection.addMarker(markerOptions);
markerCollection.setOnMarkerDragListener(new GoogleMap.OnMarkerDragListener( ... ...
```

**Comment by wangela** [comment]
If you would like to upvote the priority of this issue, please comment below or react on the original post above with :+1: so we can see what is popular when we triage.
--------

@gpopac Thank you for opening this issue. 🙏
Please check out these other resources that might help you get to a resolution in the meantime:

- [Check the issue tracker](https://developers.google.com/maps/support/#issue_tracker) - bugs and feature requests for Google Maps Platform APIs and SDKs
- [Open a support case](https://developers.google.com/maps/support#contact-maps-support) - Get 1:1 support in Cloud Console.
- [Discord](https://discord.gg/fHTDmZe9En) - chat with other developers
- [StackOverflow](http://stackoverflow.com/questions/tagged/google-maps) - use the `google-maps` tag


_This is an automated message, feel free to ignore._

**Comment by gpopac** [comment]
Update. It looks like the dragging stops working as soon as I add KmlLayer or GeoJsonLayer.
